### DISCUSSION TITLE
{"translate":"multiplayer.disconnect.invalid_public_key_signature"} false

### DISCUSSION THREAD

**Discussion by stnedi** [PrismarineJS/mineflayer#2852] [Q&A]
Hi,

New to Mindflayer and would like to experiment with bots on my MineOS hosted 1.19.2  vanilla server but I am failing at the first hurdle.

My script is identical to the Echo example with the IP address, username, password and auth type set appropriately.

How can I troubleshoot this error message?

{"translate":"multiplayer.disconnect.invalid_public_key_signature"} false

Thanks

stndi

**Comment by stnedi** [comment]
Is 1.19.2 not fully supported?  Changing the client and server to 1.18.1 allows the bot to connect.

Stnedi

**Comment by LeonKohli** [comment]
Currently I have the same issue

**Comment by graille** [comment]
Same. Anyone know why this error is raised ?

**Reply by graille** [reply]
Edit: seems adding `disableChatSigning: true` resolve the issue

**Reply by stnedi** [reply]
I will try this now and post the results.

**Reply by stnedi** [reply]
I am still seeing the same error.  Having not used disableChatSigning before, am I using it correctly?

const bot = mineflayer.createBot({
  host: 'SERVERIP', // minecraft server ip
  username: 'EMAILADDRESS', // minecraft username
  password: 'PASSWORD', // minecraft password, comment out if you want to log into online-mode=false servers
  //port: 25565,                // only set if you need a port that isn't 25565
  // version: false,             // only set if you need a specific version or snapshot (ie: "1.8.9" or "1.16.5"), otherwise it's set automatically
  auth: 'microsoft',            // only set if you need microsoft auth, then set this to 'microsoft'
  disableChatSigning: true
})

**Reply by ihatekids10203** [reply]
That script is not up-to-date. Although it would probably not make a difference, heres the correct one:

const mineflayer = require ('mineflayer')

const bot = mineflayer.createBot({
    host: "ip", 
    port: "port",
    username: "email",
    password: "password",
    auth: "microsoft",
})
